osdep/win32/pthread.c

```diff
@@ -92,15 +91,38 @@ static int cond_wait(pthread_cond_t *restrict cond,
     return res ? 0 : ETIMEDOUT;
 }
 
+static struct timespec get_realtime(void)
+{
+    // same as mpv's code
```

Comment:
Why not implement `clock_gettime` instead of duplicating this in both places?

Reply:
Sounds like a good idea at this point.